hc/api/models.py

```diff
@@ -541,6 +541,18 @@ class Ping(models.Model):
     rid = models.UUIDField(null=True)
 
     def to_dict(self) -> PingDict:
+        if self.has_body():
+            body_url = (
+                settings.SITE_ROOT
+                + "/api/v1/checks/"
+                + str(self.owner.code)
+                + "/pings/"
+                + str(self.n)
+                + "/body/"
+            )
```

Comment:
I would prefer to construct the URL using `django.urls.reverse()`.

Reply:
Fixed in https://github.com/healthchecks/healthchecks/pull/737/commits/f9cdf9ec2a716bd86b007824e7bfdbc6aa2ad1fe